@oclif/plugin-help should be a production dependency

Similar to https://github.com/oclif/command/issues/59

I have this error when using pnpm and passing `--help` to my CLI project:

```
Error: Cannot find module '@oclif/plugin-help'
```

My workaround for now is to declare `@oclif/plugin-help` as a dependency of my project.

--- a/package.json
+++ b/package.json
@@ -5,8 +5,10 @@
   "author": "Jeff Dickey @jdxcode",
   "bugs": "https://github.com/oclif/command/issues",
   "dependencies": {
+    "@oclif/config": "^1",
     "@oclif/errors": "^1.2.2",
     "@oclif/parser": "^3.8.3",
+    "@oclif/plugin-help": "^2",
     "debug": "^4.1.1",
     "semver": "^5.6.0"
   },

